hudi-common/src/main/java/org/apache/hudi/common/util/FSUtils.java

```diff
@@ -532,6 +532,13 @@ public static Path getPartitionPath(Path basePath, String partitionPath) {
     return ((partitionPath == null) || (partitionPath.isEmpty())) ? basePath : new Path(basePath, partitionPath);
   }
 
+  /**
+   * Get file URI path (e.g. hdfs://ip-address:8020:/<file absolute path>)
+   */
+  public static String getURIPath(FileSystem fs, Path filePath) {
```

Comment:
**[Trivial]** `filePath =>  path` as it is more generic. Also what you are actually passing to this function is not a file's path, but a partitions path.